addons/website_slides/models/slide_slide.py

```diff
@@ -375,16 +375,16 @@ def _compute_embed_counts(self):
         for slide in self:
             slide.embed_count = mapped_data.get(slide.id, 0)
 
-    @api.depends('slide_ids.sequence', 'slide_ids.slide_category', 'slide_ids.is_published', 'slide_ids.is_category')
+    @api.depends('slide_ids.sequence', 'slide_ids.slide_category', 'slide_ids.is_published', 'slide_ids.is_category', 'slide_ids.completion_time', 'slide_ids.active')
```

Comment:
Probably don't need to depend on "active" ?

Reply:
we do need the dependance on "active", it may not show when simply adding slides to the category, but if you delete one of the categories and then try to add new ones time computation will get all weird without it.